piper/ledspage.py

```diff
@@ -57,6 +57,12 @@ def _init_ui(self):
             mousemap.add(button, "#led{}".format(index))
             sizegroup.add_widget(button)
 
+    def _find_active_profile(self):
+        # Finds the active profile, which is guaranteed to be found.
+        for profile in self._device.profiles:
+            if profile.is_active:
+                return profile
```

Comment:
I was googling around and found this construct, figured I'd share:
````python
return next(p for p in self._device.profiles if p.is_active)
````
Also takes a default parameter, so could be used with 
````python
return next((p for p in self._device.profiles if p.is_active), self._device.profiles[0])
````
Just sharing this, I hadn't known about it :)